gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/api/ApiPlansResource.java

```diff
@@ -181,9 +187,10 @@ public Response updateApiPlan(@PathParam("plan") String plan, @Valid @NotNull Up
     @Permissions({ @Permission(value = RolePermission.API_PLAN, acls = { RolePermissionAction.DELETE }) })
     public Response deleteApiPlan(@PathParam("plan") String plan) {
```

Comment:
The path param should follow the same format as others, and be suffixed by `Id`
```suggestion
    public Response deleteApiPlan(@PathParam("planId") String planId) {
```